### PR TITLE
add_support_for_multiple_dockers_in_lint

### PR DESCRIPTION
**Pull request by YuvHayun**
## Related Issues
fixes:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+* added support for multiple dockers separated by comma for native docker linter.
```
Comment (dorschw) [inline CHANGELOG.md:4]:
```suggestion
* added support for running **lint** in multiple native-docker images.
```

File: demisto_sdk/commands/lint/linter.py [hunk 2]
```diff
@@ -314,11 +314,15 @@ def _gather_facts(self, modules: dict) -> bool:
...
         if self._facts["docker_engine"]:
             logger.info(f"{log_prompt} - Collecting all docker images to pull")
-            images = self._get_docker_images_for_lint(
-                script_obj=script_obj,
-                script_id=yml_obj_id,
-                docker_image_flag=self.docker_image_flag,
-            )
+            docker_images = self.docker_image_flag.split(",")
+            images = []
+            for docker_image in docker_images:
```
Comment (dorschw) [inline demisto_sdk/commands/lint/linter.py:319]:
```suggestion
            images = []
            for docker_image in self.docker_image_flag.split(","):
```

**Review by dorschw — APPROVED** [review]

File: demisto_sdk/__main__.py [hunk 3]
```diff
@@ -922,7 +922,7 @@ def secrets(config, **kwargs):
     "-di",
     "--docker-image",
     default="from-yml",
-    help="The docker image to check package on. Possible values: 'native:maintenance', 'native:ga', 'native:dev',"
+    help="The docker image to check package on. A comma separated list of Possible values: 'native:maintenance', 'native:ga', 'native:dev',"
```
Comment (JudahSchwartz) [inline demisto_sdk/__main__.py:925]:
```suggestion
    help="The docker image to check package on. Can be a comma separated list. Possible values: 'native:maintenance', 'native:ga', 'native:dev', 'native:dev,native:ga',"
```

File: demisto_sdk/commands/lint/linter.py [hunk 4]
```diff
@@ -314,11 +314,15 @@ def _gather_facts(self, modules: dict) -> bool:
...
+                temp_images = self._get_docker_images_for_lint(
+                    script_obj=script_obj,
+                    script_id=yml_obj_id,
+                    docker_image_flag=docker_image,
+                )
+                images.extend(temp_images)
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/lint/linter.py:325]: lets make sure we turn to set at some point to make sure theres no duplicates

**Pull request merged**